src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/TalismanListener.java

```diff
@@ -292,6 +296,35 @@ public void onBlockBreak(BlockBreakEvent e) {
         }
     }
 
+    @EventHandler
+    public void onExperienceReceive(PlayerExpChangeEvent e) {
+        if (e.getAmount() <= 0 || !Talisman.checkFor(e, SlimefunItems.TALISMAN_WISE)) {
+            return;
+        }
+
+        e.setAmount(e.getAmount() * 2);
```

Comment:
I think the `return;` here really serves no point other than double-negation.
I reckon it would be more understandable if you just did
```suggestion
        if (e.getAmount() > 0 && Talisman.checkFor(e, SlimefunItems.TALISMAN_WISE)) {
            e.setAmount(e.getAmount() * 2);
        }
```

Reply:
Im used to this kind of style (check if everything is correct, if not return - used to do this to avoid nesting a lot), so I did it out of habit

Reply:
If used excessively this can get worse than nesting, especially since this would just be a one-level nesting.